audit_exceptions/mismatched_binary_allowlist.json

```
@@ -0,0 +1,3 @@
+[
+  "btb/open-watcom/open-watcom-v2"
```

```suggestion
  "open-watcom-v2"
```

This is what the exception list should look like. The full name shouldn't be used. Just to make sure, the reason this didn't work is because of an error like this, right?

```console
$ brew audit open-watcom-v2
btb/open-watcom:
  * audit_exceptions/mismatched_binary_allowlist.json references
    formulae or casks that are not found in the btb/open-watcom tap.
    Invalid formulae or casks: open-watcom-v2
```

Assuming yes, this is a bug in Homebrew/brew. I've got a fix ready

---

Yep, that's the error: https://github.com/Homebrew/discussions/discussions/2005#discussioncomment-1203817

Thanks for having a look @Rylan12!

---

👍 Fix is in https://github.com/Homebrew/brew/pull/11895. Once that's merged this can be rerun and should work fine when using the short formula name in the list.

---

Fix is merged; try applying this suggestion again.